test/verify/check-machines

```diff
@@ -2478,12 +2488,34 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
             self._assertScriptFinished() \
                 .checkEnvIsEmpty()
 
+        def checkErrorNotification(self, error_msg):
+            self.browser.wait_present("#app div[aria-label=\"Danger Alert\"]")
+            self.browser.click("#app button.alert-link.more-button.link-button")
+            self.browser.wait_in_text("#app div.pf-c-alert__description", error_msg)
+
         def installWithErrorTest(self, dialog):
             self._tryInstallWithError(dialog) \
                 ._assertScriptFinished() \
                 ._deleteVm(dialog) \
                 .checkEnvIsEmpty()
 
+        def createThenDeleteInInstallation(self, dialog, error_msg):
+            name = dialog.name
+
+            dialog.open().fill()
+            self._create(dialog)
+
+            if not dialog.start_vm:
+                self.browser.click("#vm-{}-install".format(name))
+                self._assertVmStates(name, "creating VM installation", "running")
+
+            self.browser.click("#vm-{}-delete".format(name))
```

Comment:
There is also _deleteVm method - would be nice to reuse - but since otherwise looks good I will merge it as is.